README.md

```diff
@@ -146,3 +146,18 @@ over the set of tested equations and some specific examples may differ.
 ### Stiff DDEs
 
 - The Rosenbrock methods, specifically `Rodas5`, perform well.
+
+### Parameter Estimation
+
+- Broadly two different approaches have been used, Bayesian Inference and Optimisation 
+  algorithms.
+- In general it seems that the optimisation algorithms perform more accurately but that can be 
+  attributed to the larger number of data points being used in the optimisation cases, Bayesian 
+  approach tends to be slower of the two and hence lesser data points are used, accuracy can 
+  increase if proper data is used.
+- Within the different available optimisation algorithms, BBO, GN_CRS2_LM, LD_SLSQP, LN_BOBYQA 
```

Comment:
Name which packages these are from, and mention the difference between global and local.